lib/datadog/appsec/waf/errors.rb

```diff
@@ -0,0 +1,35 @@
+module Datadog
+  module AppSec
+    module WAF
+      class Error < StandardError
```

Comment:
We can just declare without opening

```suggestion
      Error = Class.new(StandardError)
```